src/app/modules/dashboard/components/beacon-node-status/beacon-node-status.component.html

```diff
@@ -1,31 +1,31 @@
-<mat-card class="bg-paper" *ngIf="(nodeConnection$ | async) as conn">
+<mat-card class="bg-paper">
   <div class="flex items-center">
     <div>
       <div
         class="pulsating-circle"
-        [class.green]="conn.connected"
-        [class.red]="!conn.connected"></div>
+        [class.green]="(beaconState.connected$ | async)"
```

Comment:
Seems like the logic became more verbose than just doing a single async pipe at the beginning

Reply:
The Idea was that the beaconNodeState is just a container for all the different informations ... if you just need the connected state somewhere, you could get it with this.nodeService.beaconNodeState.connected$ 
Otherwise you would always have to update the whole container object ... with this sollution you could have different intervals and only update specific infortmaions.